online-shop/src/app/modules/shopping-cart/components/containers/shopping-cart-details/shopping-cart-details.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { mockProducts } from 'src/app/mocks/products.mocks';
+import { Product } from 'src/app/modules/shared/types/products.types';
+
+@Component({
+  selector: 'app-shopping-cart-details',
+  templateUrl: './shopping-cart-details.component.html',
+  styleUrls: ['./shopping-cart-details.component.scss']
+})
+export class ShoppingCartDetailsComponent {
+  products: Product[] = mockProducts;
```

Comment:
I suggest you make an interface called ProductWithQuantities and when you initialize the quantities, you should do something like `this.productsWithQuantities = this.products.map(product => ({ ...product, quantity: 0 }));`

The spread operator '...' takes all the properties of the initial object and quantity: 0 declares a new property initialized with 0. The structure () => ({ }) creates a new object.